assignments/arrays.js

```diff
@@ -63,34 +63,122 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
```

Comment:
great job on adding the comments to explain your code!!!! 

I love that your code is dynamic so you can answer the required question but also any question a boss could ask about any car in the dataset, even if the dataset is updated.